basic-conditionals.tex

```diff
@@ -0,0 +1,400 @@
+
+\begin{Verbatim}[commandchars=\\\{\}]
+\PY{c+cp}{\PYZsh{}}\PY{c+cp}{include} \PY{c+cpf}{\PYZlt{}stdio.h\PYZgt{}}
+
+\PY{k+kt}{int} \PY{n+nf}{main}\PY{p}{(}\PY{p}{)} \PY{p}{\PYZob{}}
+    \PY{k+kt}{int} \PY{n}{a}\PY{p}{,} \PY{n}{b}\PY{p}{;}
+    \PY{n}{scanf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{\PYZpc{}d\PYZpc{}d}\PY{l+s}{\PYZdq{}}\PY{p}{,} \PY{o}{\PYZam{}}\PY{n}{a}\PY{p}{,} \PY{o}{\PYZam{}}\PY{n}{b}\PY{p}{)}\PY{p}{;}
+        
+    \PY{k}{if} \PY{p}{(}\PY{n}{a} \PY{o}{=}\PY{o}{=} \PY{n}{b}\PY{p}{)} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Stevilki sta enaki.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{p}{\PYZcb{}}
+    \PY{k}{if} \PY{p}{(}\PY{n}{a} \PY{o}{!}\PY{o}{=} \PY{n}{b}\PY{p}{)} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Stevilki sta razlicni.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{p}{\PYZcb{}}
+    \PY{k}{if} \PY{p}{(}\PY{n}{a} \PY{o}{\PYZlt{}} \PY{n}{b}\PY{p}{)} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Prva stevilka je manjsa od druge.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{p}{\PYZcb{}}
+    \PY{k}{if} \PY{p}{(}\PY{n}{a} \PY{o}{\PYZgt{}} \PY{n}{b}\PY{p}{)} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Prva stevilka je vecja od druge.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{p}{\PYZcb{}}
+    \PY{k}{if} \PY{p}{(}\PY{n}{a} \PY{o}{\PYZlt{}}\PY{o}{=} \PY{n}{b}\PY{p}{)} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Prva stevilka je manjsa ali enaka drugi.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{p}{\PYZcb{}}
+    \PY{k}{if} \PY{p}{(}\PY{n}{a} \PY{o}{\PYZgt{}}\PY{o}{=} \PY{n}{b}\PY{p}{)} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Prva stevilka je vecja ali enaka drugi.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{p}{\PYZcb{}}
+    \PY{k}{return} \PY{l+m+mi}{0}\PY{p}{;}
+\PY{p}{\PYZcb{}}
+\end{Verbatim}
+
+\begin{inout}
+12 12
+\tcblower
+Stevilki sta enaki.\\
+Prva stevilka je manjsa ali enaka drugi.\\
+Prva stevilka je vecja ali enaka drugi.
+\end{inout}
+
+\begin{inout}
+3 7
+\tcblower
+Stevilki sta razlicni.\\
+Prva stevilka je manjsa od druge.\\
+Prva stevilka je majnsa ali enaka drugi.
+\end{inout}
+
+\end{examples}
+
+
+\begin{errors}
+  Pri primerjavi moramo uporabiti dva enačaja (\verb+==+). Če uporabimo samo en enačaj, kot v matematiki (torej \verb+=+), se bo program sicer zagnal, vendar ne bo deloval pravilno.
+  Enojni enačaj uporabljamo samo pri nastavljanju vrednosti neki spremenljivki, in nikoli v običajnih oklepajih v \verb+if+ stavku.
+\end{errors}
+
+\begin{errors}
+  Taka primerjava deluje samo na številkah. Če želimo primerjati med dvema besediloma, za to potrebujemo posebno funkcijo \verb+strcmp+, ki jo bomo podrobneje spoznali, ko bomo govorili o besedilu.
```

Comment:
Jaz sem malo v dvomih glede omenjanja `strcmp`. Po eni strani je fino, ker bodo imeli to za referenco potem ko bodo znali `strcmp`, po drugi strani pa nocmo da bi jih to iztirilo zdaj ko njega se ne poznajo.

Reply:
Oni to poznajo, ker smo omenili na predavanjih. Sem dodal komentar, da je smiselno premisliti, če se to ohrani tudi v učbeniku.

Reply:
aha. ne vem. js se vseeno nagibam k temu da ne bi dal tega, ampak sm pa zdej manj nenaklonjen ko to vem